deployment/README.md

```diff
@@ -1,379 +1,466 @@
+    - `manual-mobile-app-appid` is the value of the `App ID` that you configured, or will configure, on the Settings page of the [`Study builder`](/study-builder/)
+    - `manual-android-bundle-id` is the value of `applicationId` that you configured in [`Android/app/build.gradle`](/Android/app/build.gradle)
+    - `manual-fcm-api-url` is the URL of your Firebase Cloud Messaging API
+    - `manual-android-server-key` is your Firebase Cloud Messaging server key
+    - `manual-android-deeplink-url` is the URL to redirect to after Android login
+    - `manual-ios-bundle-id` is the value you obtained from Xcode
+    - `manual-ios-certificate` is the value of the Base64 converted `.p12` file
+    - `manual-ios-certificate-password` is the value of the password for the `.p12` certificate 
+    - `manual-ios-deeplink-url` is the URL to redirect to after iOS login
+1. Initialize your `Participant datastore` database to work with your mobile applications
+    - Once a study with the `App ID` corresponding to the `manual-mobile-app-appid` secret is created and published using the [`Study builder`](/study-builder) user interface, a corresponding 
+app record will appear in the [`Participant manager`](/participant-manager/) user interface (if you created a study before all of your platform components were operational, you can reinitialize this process by using the `Study builder` user interface to pause and resume the study) 
+    - Once the `App ID` appears in `Participant manager`, run the [deployment/scripts/copy_app_info_to_sql.sh](/deployment/scripts/copy_app_info_to_sql.sh) script to update the remaining databases, for example:
+         ```bash
+         $GIT_ROOT/deployment/scripts/copy_app_info_to_sql.sh $PREFIX $ENV
+         ```
+
+### Clean up
+
+1. Remove your user account from the groups you no longer need access to
+1. Revoke user access in your environment, for example:
     ```bash
-    # bundleID used for the Android App.
-    manual-android-bundle-id
-    # found under settings > cloud messaging in the android app defined in your firebase project.
-    manual-android-server-key
-    # bundleID used to build and distribute the iOS App.
-    manual-ios-bundle-id
-    # push notifications certificate in encrypted .p12 format.
-    manual-ios-certificate
-    # push notifications certificate password.
-    manual-ios-certificate-password
-    # redirect links to mobile apps, e.g. app://mydeploymentdomain.com/mystudies
-    manual-ios-deeplink-url
-    manual-android-deeplink-url
+    gcloud auth revoke --all -q && \
+      gcloud auth application-default revoke -q
     ```
+<!--
+### Troubleshooting
 
-### Step 13: Clean up
-
-1. Revoke your super admin access by running `gcloud auth revoke` and
-    authenticate as a normal user for daily activities.
+See the [*Troubleshooting*](/documentation/troubleshooting.md) guide for more information.
+--->
```

Comment:
let's just remove this part instead of leaving it commented out

Reply:
will do